src/common/ceph_time.h

```diff
@@ -20,6 +20,19 @@
 
 #include "include/encoding.h"
 
+#if defined(DARWIN)
+#include <sys/_types/_timespec.h>
+#include <mach/mach.h>
+#include <mach/clock.h>
+
+#define CLOCK_REALTIME CALENDAR_CLOCK
```

Comment:
@ukernel Perhaps put this in include/compat.h like I do for FreeBSD. That way all clock compatability stuff is "hidden" in there.